csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/SourceGenerators/ResxGenerator.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using Semmle.Util;
+using Semmle.Util.Logging;
+
+namespace Semmle.Extraction.CSharp.DependencyFetching
+{
+    internal class ResxGenerator : DotnetSourceGeneratorBase<Resx>
+    {
+        private readonly string? sourceGeneratorFolder = null;
+
+        public ResxGenerator(
+            FileProvider fileProvider,
+            FileContent fileContent,
+            IDotNet dotnet,
+            ICompilationInfoContainer compilationInfoContainer,
+            ILogger logger,
+            NugetPackageRestorer nugetPackageRestorer,
+            TemporaryDirectory tempWorkingDirectory,
+            IEnumerable<string> references) : base(fileProvider, fileContent, dotnet, compilationInfoContainer, logger, tempWorkingDirectory, references)
+        {
+            try
+            {
+                // The package is downloaded to `missingpackages`, which is okay, we're already after the DLL collection phase.
+                var nugetFolder = nugetPackageRestorer.TryRestore("Microsoft.CodeAnalysis.ResxSourceGenerator");
```

Comment:
This also takes the "newest" package folder based on lexicographic ordering. Is that acceptable for this package (I don't know their version numbers)?